epochStart/metachain/rewards.go

```diff
@@ -262,7 +262,11 @@ func (r *rewardsCreator) CreateMarshalizedData(body *block.Body) map[string][][]
 }
 
 // SaveTxBlockToStorage saves created data to storage
-func (r *rewardsCreator) SaveTxBlockToStorage(metaBlock *block.MetaBlock, body *block.Body) {
+func (r *rewardsCreator) SaveTxBlockToStorage(
```

Comment:
Put the miniblockpool on the constructor. Of the component

Reply:
done